autogen/oai/client.py

```diff
@@ -314,6 +327,8 @@ class OpenAIWrapper:
         "api_version",
         "api_type",
         "tags",
+        "window_size",
```

Comment:
are we doing anything with window size?

Reply:
Yes. Both :)

I have yet to modify WebSurfer (this is still a raft PR), but it will use WindowSize to understand how to break up pages. 

I also use it in the complex_tasks branch with SocietyOfMind and will port those changes to main once this is ready and merged.